tensorflow_addons/optimizers/tests/standard_tests.py

```diff
@@ -18,7 +18,7 @@
 import tensorflow as tf
```

Comment:
Why the file name is changed?

Reply:
I think I changed it because there would be multiple tests? We can change it back?

Reply:
Let's change it back. I think it's better to have a standard naming convention for file!